src/graphql/resolvers/postOfferResolver.test.js

```diff
@@ -1,40 +1,130 @@
+import PostOfferResolver from "./postOfferResolver";
+
 describe("Resolvers", () => {
   it("PostOfferResolver", async () => {});
 });
 
-// const testcases = [
-//   {
-//     desc: "should return student object with the given args and a uuid",
-//     db: {
-//       put: jest.fn().mockReturnValue({ promise: () => {} })
-//     },
-//     args: {
-//       student: {
-//         firstname: "Braden",
-//         lastname: "Watkins"
-//       }
-//     },
-//     expectedDBCall: {
-//       TableName: "Student",
-//       Item: {
-//         firstname: "Braden",
-//         lastname: "Watkins",
-//         id: uuidv4()
-//       }
-//     },
-//     expectedRetValue: {
-//       firstname: "Braden",
-//       lastname: "Watkins",
-//       id: uuidv4()
-//     }
-//   }
-// ];
+const uuidv4 = require("uuid/v4");
```

Comment:
This could be an es6 import statement

Reply:
I think I just copied that from the PostOfferResolver. I will go ahead and change it there also.